terraform/00_state.tf

```diff
@@ -16,22 +16,28 @@
 # backend which stores everything in a local directory. The config you see below
 # is functionally equivalent to the defaults.
 
-terraform {
-  backend "local" {
-    path = "terraform.tfstate"
-  }
-}
+#terraform {
+#  backend "local" {
+#    path = "terraform.tfstate"
+#  }
+#}
 
 # TODO - Consider storing state remotely on a per-instance basis. 
 # In that case we'd generate a backend config just before
 # running terraform that would look something like this:
 
-# terraform {
-#   backend "gcs" {
-#     bucket = "stackdriver-sandbox"
-#     prefix = "<project-id>"
-#   }
-# }
+terraform {
+   backend "gcs" {
+       bucket="stackdriver-sandbox-1882647097-bucket"
```

Comment:
What is the idea behind **1882647097** in the name of the bucket?

Reply:
This is a mistake. I removed it.